Assets/Scripts/EnemyShooting4Guns.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyShooting4Guns : MonoBehaviour
+{
+
```

Comment:
```suggestion
     [SerializeField] private int gunAmount;
```
Hier mee kan je zeggen hoeveel guns de enemy heeft, dan kan je de andere 2 script weg doen.